torchaudio/datasets/speechcommands.py

```diff
@@ -79,9 +92,22 @@ def __init__(self,
                     download_url(url, root, hash_value=checksum, hash_type="md5")
                 extract_archive(archive, self._path)
 
-        walker = walk_files(self._path, suffix=".wav", prefix=True)
-        walker = filter(lambda w: HASH_DIVIDER in w and EXCEPT_FOLDER not in w, walker)
-        self._walker = list(walker)
+        def load_list(filename):
+            filepath = os.path.join(self._path, filename)
+            with open(filepath) as f:
+                return [os.path.join(self._path, l.strip()) for l in f]
```

Comment:
> Please prefer a plain function defined on module level especially when there is no local variable needs to be captured.

`load_list` uses `self._path` in its current form :)

> Also, since you are defining a helper function for your own sake, you can adopt a signature like
> 
> ```python
> def load_lists(*filename)
> ```
> 
> and it will make `excludes = load_list(VALIDATION_LIST) + load_list(TESTING_LIST)` into as simple as `excludes = load_list("validation_list.txt", "testing_list.txt")`. (continue to the next comment)

Either way looks reasonable to me, but I've made the update since you have a preference :)

Reply:
> `load_list` uses `self._path` in its current form :)

Oops my bad.